ocaml-lsp-server/test/e2e/__tests__/textDocument-codeAction.test.ts

```diff
@@ -66,6 +66,30 @@ let f (x : t) = x
         kind: "destruct",
         title: "Destruct",
       },
+      {
+        edit: {
+          changes: {
+            "file:///test.ml": [
+              {
+                newText: "(x : type t = Foo of int | Bar of bool)",
```

Comment:
here is the place where the type is not displayed as it should

Reply:
Yes, you should only enable this code actions on expressions.

Reply:
Would querying the type of an expression using mconfig verbosity = 0 (see `ocaml_lsp_server/src/document.ml:136`) help here?

Then we go from 

```ocaml
let (not_supported : ('a, Jsonrpc.Response.Error.t) result

type ('a, 'b) result = Ok of 'a | Error of 'b) =
  Error
    (make_error ~code:InternalError ~message:"Request not supported yet!" ())
```

to correct

```ocaml
let (not_supported : ('a, Jsonrpc.Response.Error.t) result) =
  Error
    (make_error ~code:InternalError ~message:"Request not supported yet!" ())
```

Reply:
Thanks, I found a way to reduce the verbosity and it seems to help